frontend/rolecall/src/app/services/super-ballet-display.service.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from '@angular/core';
+
+
+@Injectable({providedIn: 'root'})
+export class SuperBalletDisplayService {
+  superBalletIsOpenMap = new Map<string, boolean>();
+
+  changeOpenState(uuid: string, isOpen: boolean) {
+    this.superBalletIsOpenMap.set(uuid, isOpen);
+  }
+
+  isOpen(uuid: string): boolean {
+    return Boolean(this.superBalletIsOpenMap.get(uuid));
+  }
+
+  removeFromDisplayList(uuid: string) {
+    if (this.superBalletIsOpenMap.has(uuid)) {    
```

Comment:
I don't think you need the if() check here, it can probably be just
```
removeFromDisplayList(balletId: string) {
  this.superBalletIsOpenMap.delete(balletId);
}
```

According to [the docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/delete), delete() will just return false if the prop didn't exist.

Reply:
Done.